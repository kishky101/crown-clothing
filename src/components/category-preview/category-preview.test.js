import renderer from 'react-test-renderer';
import CategoryPreview from './category-preview.components';
import 'jest-styled-components';
import { MockStore } from '../../utils/test/setup';
import { BrowserRouter as Router } from 'react-router-dom';


test('creating category-preview snapshot', () => {
        const carticon = renderer.create(
            <MockStore>
                <Router>
                    <CategoryPreview title='hats' products={[{
                        id: 6,
                        name: 'Palm Tree Cap',
                        imageUrl: 'https://i.ibb.co/rKBDvJX/palm-tree-cap.png',
                        price: 14,
                        quantity: 2
                    }]} />
                </Router>
            </MockStore>
            
        ).toJSON;
        expect(carticon).toMatchSnapshot();
})