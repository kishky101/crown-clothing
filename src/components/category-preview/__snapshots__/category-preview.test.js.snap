// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating category-preview snapshot 1`] = `[Function]`;
